src/Sylius/Bundle/ShopBundle/templates/checkout/address/form/addresses.html.twig

```diff
@@ -0,0 +1,7 @@
+{% set channel = hookable_metadata.context.order.channel %}
+
+{% if channel.shippingAddressInCheckoutRequired %}
+    {% include '@SyliusShop/checkout/address/form/addresses/shipping_address_required.html.twig' %}
+{% else %}
+    {% include '@SyliusShop/checkout/address/form/addresses/billing_address_required.html.twig' %}
```

Comment:
These includes should be converted into Twig Hooks, and ideally all address fields should be divided into separate templates so that this can be customized as easily as possible